nfConfigApi/api_default.go

```diff
@@ -418,6 +419,112 @@ func (a *DefaultAPIService) NfconfigPolicyControlGetExecute(r ApiNfconfigPolicyC
 	return localVarReturnValue, localVarHTTPResponse, nil
 }
 
+type ApiNfconfigQosDnnImsiGetRequest struct {
+	ctx        context.Context
+	ApiService *DefaultAPIService
+	dnn        string
+	imsi       string
+}
+
+func (r ApiNfconfigQosDnnImsiGetRequest) Execute() ([]ImsiQos, *http.Response, error) {
+	return r.ApiService.NfconfigQosDnnImsiGetExecute(r)
+}
+
+/*
+NfconfigQosDnnImsiGet Get QoS for given IMSI and DNN
+
+	@param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
+	@param dnn Data Network Name
+	@param imsi IMSI in format “imsi-” followed by 15 or 16 digits
+	@return ApiNfconfigQosDnnImsiGetRequest
+*/
+func (a *DefaultAPIService) NfconfigQosDnnImsiGet(ctx context.Context, dnn string, imsi string) ApiNfconfigQosDnnImsiGetRequest {
+	return ApiNfconfigQosDnnImsiGetRequest{
+		ApiService: a,
+		ctx:        ctx,
+		dnn:        dnn,
+		imsi:       imsi,
+	}
+}
+
+// Execute executes the request
+//
+//	@return []ImsiQos
+func (a *DefaultAPIService) NfconfigQosDnnImsiGetExecute(r ApiNfconfigQosDnnImsiGetRequest) ([]ImsiQos, *http.Response, error) {
+	var (
+		localVarHTTPMethod  = http.MethodGet
+		localVarPostBody    interface{}
+		formFiles           []formFile
+		localVarReturnValue []ImsiQos
+	)
+
+	localBasePath, err := a.client.cfg.ServerURLWithContext(r.ctx, "DefaultAPIService.NfconfigQosDnnImsiGet")
+	if err != nil {
+		return localVarReturnValue, nil, &GenericOpenAPIError{error: err.Error()}
+	}
+
+	localVarPath := localBasePath + "/nfconfig/qos/{dnn}/{imsi}"
+	localVarPath = strings.Replace(localVarPath, "{"+"dnn"+"}", url.PathEscape(parameterValueToString(r.dnn, "dnn")), -1)
```

Comment:
What about doing something like this instead?
```suggestion
	localVarPath = strings.ReplaceAll(localVarPath, "{"+"dnn"+"}", url.PathEscape(parameterValueToString(r.dnn, "dnn")))
```
Comment also applies to the next line

Reply:
This can be done through `templates` but I think we can just leave it as-is. Thanks!

Reply:
Since you added templates, thos comments can be addresses now, correct?